source/retryable-writes/tests/insertOne-errorLabels.yml

```diff
@@ -0,0 +1,44 @@
+runOn:
+    - minServerVersion: "4.3.x"
+      topology: ["replicaset", "sharded"]
+
+data: []
+
+tests:
+    - description: "InsertOne succeeds after RetryableWriteError from server"
+      failPoint:
+          configureFailPoint: failCommand
+          mode: { times: 1 }
+          data:
+              failCommands: ["insert"]
+              errorLabels: ["RetryableWriteError"]
+              errorCode: 112 # WriteConflict, not a retryable error code
+          operation:
+              name: "insertOne"
+              arguments:
+                  document: { _id: 1, x: 11 }
+          outcome:
+              result:
+                  insertedId: 1
+              collection:
+                  data:
+                      - { _id: 1, x: 11 }
+
+    - description: "InsertOne fails if server does not return RetryableWriteError"
+        failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands: ["insert"]
+                errorLabels: []
+                errorCode: 11600 # InterruptedAtShutdown, normally a retryable error code
+            operation:
+                name: "insertOne"
+                arguments:
+                  document: { _id: 1, x: 11 }
+          outcome:
+            error: true
+            result:
+                errorLabelsOmit: ["RetryableWriteError"]
```

Comment:
Here we test that the label is omitted. I think that justifies also testing for omission in the similar abort/commit tests below.